Pipfile

```diff
@@ -24,17 +24,22 @@ boto3 = "*"
 pexpect = "*"
 pysftp = "*"
 setuptools = "*"
-cryptography = "==3.3.2"
+cryptography = "==3.4.8"
 sqlalchemy = "==1.3.23"
 flask-openid = "==1.3.0"
 flask-sqlalchemy = "==2.4.4"
 jinja2 = "==2.11.3"
 werkzeug = "==0.16.1"
 marshmallow = "==2.21.0"
 wtforms = "==2.3.3"
+markupsafe= "==2.0.1"
```

Comment:
I was getting an error where c_types are not defined.  This is a known issue that affects all markupsafe after this version.  Flaask and Jinja use this.